xml/admin_monitoring_alerting.xml

```diff
@@ -366,36 +366,102 @@ placement:
+    <replaceable>ADD_PLACEMENT_HERE</replaceable>
+image: registry.suse.com/ses/7/prometheus-webhook-snmp:latest
+args:
+    - "--publish 9099:9099"
+envs:
+    - ARGS="--debug --snmp-host=<replaceable>ADD_HOST_GATEWAY_HERE</replaceable>"
+    - RUN_ARGS="--metrics"
+EOF
+</screen>
+     <para>
+      This service specification is used to get the service running using
+      its default settings.
+     </para>
+     <para>
+      You need to publish the port the &prometheus; receiver is listening on by
+      using the command line argument <literal>--publish <replaceable>HOST_PORT</replaceable>:<replaceable>CONTAINER_PORT</replaceable></literal>
+      when running the service because the port is not exposed automatically by the
+      container. This can be done by adding the following lines to the specification:
+     </para>
+<screen>
+args:
+    - "--publish 9099:9099"
+</screen>
+     <para>
+       Alternatively, connect the container to the host network by using
+       the command line argument <literal>--network=host</literal>.
+     </para>
 <screen>
-monitoring:
- alertmanager:
-   receiver:
-      snmp:
-        enabled: True
-        config:
-          host: localhost
-          port: 9099
-          snmp_host: snmp.foo-bar.com
-          snmp_community: private
-          metrics: True
+args:
+    - "--network=host"
 </screen>
      <para>
-      Refer to the receiver manual at
-      <link xlink:href="https://github.com/SUSE/prometheus-webhook-snmp#global-configuration-file" />.
-      for more details about the configuration options.
+      Additionally, the SNMP host needs to be configured when the
+      container is connecting to the host network via <literal>--network=host</literal>.
+      Use the container's network gateway to be able to receive SNMP traps
+      outside the container:
      </para>
-    </example>
-  </sect1>-->
+<screen>
+envs:
+    - ARGS="--debug --snmp-host=<replaceable>CONTAINER_GATEWAY</replaceable>"
+</screen>
+     <sect2 xml:id="configure-prometheus-">
+       <title>Configure the prometheus-webhook-snmp service</title>
+       <para>
+         The container can be configured by environment variables or by
+         using a configuration file.
+       </para>
+       <para>
+         For the environment variables, use <literal>ARGS</literal> to set global options and <literal>RUN_ARGS</literal>
+         for the run command options. You need to adapt the service specification
+         the following way:
+       </para>
+<screen>
+envs:
+    - ARGS="--debug --snmp-host=<replaceable>CONTAINER_GATEWAY</replaceable>"
+    - RUN_ARGS="--metrics --port=9101"
+</screen>
+       <para>
+         To use a configuration file, the service specification must be adapted the following way:
+       </para>
+<screen>
+files:
+    etc/prometheus-webhook-snmp.conf:
+        - "debug: True"
+        - "snmp_host: <replaceable>ADD_HOST_GATEWAY_HERE</replaceable>"
+        - "metrics: True"
+        - metrics: True
```

Comment:
Should this be duplicated like this, @votdev ?

Reply:
No, only 
```
        - "debug: True"
        - "snmp_host: <replaceable>ADD_HOST_GATEWAY_HERE</replaceable>"
        - "metrics: True"
```